### PR TITLE
Three scale resume get put

### PR DESCRIPTION
**Pull request by NicholasBusby**
DO NOT MERGE!!!!
I am wanting this to get approved but not merged so that it does not cause problems.  This will be merged at the same time as the consumer-domain changes.

Add your name here if your comfortable merging
[  ]

Note 1: I have ran rubocop on all .rb files I edited.
Note 2:To test you will need to get a 3scale bearer token
Note 3: to test
```ruby
bundle exec pry
require 'Cb'
Cb.configure do |config|
  config.dev_key    = 'DEVKEYHERE'
  config.time_out  = 30
  config.debug_api  = true
end
request = Cb::Requests::Resumes::Get.new( { resume_hash: 'your_resume_hash', three_scale_token: 'your_token' } )
client = Cb::Client.new

response = client.execute request
```

### CODE REVIEW

**Comment by usbsnowcrash** [comment]
@NicholasBusby Be careful to not put dev keys in comments....technically it means we have to rotate a key every time we do that

**Comment by NicholasBusby** [comment]
Ahh sorry, I missed that one.  I wasn't think about it because it was not directly needed for the 3scale call.  Sorry.

File: lib/cb/config.rb [hunk 1]
```diff
@@ -33,12 +33,13 @@ def set_defaults
       @use_json             = true
       @host_site            = Cb.country.US
       @test_resources       = false
-      @observers            = Array.new
+      @observers            = []
```
Comment (usbsnowcrash) [inline lib/cb/config.rb]: :+1:  ruby styles

File: spec/cb/requests/resumes/delete_spec.rb [hunk 2]
```diff
@@ -34,7 +34,7 @@ module Cb
 
 
         it 'will be correctly configured' do
-          expect(request.endpoint_uri).to eq(Cb.configuration.uri_resume_get.sub(':resume_hash', 'resumeHash'))
+          expect(request.endpoint_uri).to eq(Cb.configuration.uri_resume_delete.sub(':resume_hash', 'resumeHash'))
```
Comment (ausmith) [inline spec/cb/requests/resumes/delete_spec.rb]: Why is resume delete not also getting switched over?

Comment (NicholasBusby) [inline spec/cb/requests/resumes/delete_spec.rb]: It is not included in the scope of Resume View Edit.

**Comment by NicholasBusby** [comment]
Closing this till consumer-domain is ready for it.

**Pull request closed without merging**